applications/nrf5340_audio/src/audio/audio_datapath.c

```diff
@@ -113,10 +114,12 @@ LOG_MODULE_REGISTER(audio_datapath, CONFIG_LOG_AUDIO_DATAPATH_LEVEL);
 #define DRIFT_ERR_THRESH_LOCK 16
 #define DRIFT_ERR_THRESH_UNLOCK 32
 
-// TODO: Re-enable
-#define PRES_COMP_ENABLE false
-/* Presentation delay in microseconds */
-#define PRES_DLY_US 10000
+#define PRES_COMP_ENABLE true
+
+/* Just in time defines */
```

Comment:
Should use multi line commenting like:
/*
  *
  */

Reply:
It's two separate comments though, maybe I'll just remove the first one